python/vision_explanation_methods/evaluation/pointing_game.py

```diff
@@ -131,6 +142,11 @@ def visualize_highly_salient_pixels(self,
             plt_fig_axis=(fig, ax),
             use_pyplot=False
         )
+        x, y, width, height = 247, 192, 108, 301
```

Comment:
Why is this hard-coded here? Shouldn't this be inferred from the image or passed as input to the class?

Reply:
oops!! leftover code from testing, removed